soc/xtensa/intel_adsp/ace_v1x/power.c

```diff
@@ -5,16 +5,30 @@
  */
 #include <zephyr/zephyr.h>
 #include <zephyr/pm/pm.h>
+#include <cpu_init.h>
 
 #include <ace_v1x-regs.h>
 
+#define CHECK_DELAY             256
```

Comment:
Is this a random or documented value ? It is worth some comment.

Reply:
ok, so doesn't make sense to put in a header instead of having it duplicated ? It seems a quite random value, so at least, lets keep it in on single place.